### PR TITLE
chore(css): Remove reflexbox from deploys

### PR DESCRIPTION
**Pull request by evanpurkhiser**

### CODE REVIEW

File: src/sentry/static/sentry/app/views/projectsDashboard/deploys.jsx [hunk 1]
```diff
@@ -70,66 +70,77 @@ class Deploy extends React.Component {
...
           </Button>
         </Background>
-      </DeployBox>
+      </DeploayContainer>
     );
   }
 }
 
-const DeployBox = styled(Box)`
+const DeploayContainer = styled('div')`
```
Comment (priscilawebdev) [inline src/sentry/static/sentry/app/views/projectsDashboard/deploys.jsx]:
```suggestion
const DeployContainer = styled('div')`
```

File: src/sentry/static/sentry/app/views/projectsDashboard/deploys.jsx [hunk 2]
```diff
@@ -31,7 +31,7 @@ export default class Deploys extends React.Component {
 
     if (deploys.length) {
       return (
-        <DeployBox p={2} pt={1}>
+        <DeploayContainer>
```
Comment (markstory) [inline src/sentry/static/sentry/app/views/projectsDashboard/deploys.jsx]:
```suggestion
        <DeployContainer>
```

File: src/sentry/static/sentry/app/views/projectsDashboard/deploys.jsx [hunk 3]
```diff
@@ -40,7 +40,7 @@ export default class Deploys extends React.Component {
               organization={organization}
             />
           ))}
-        </DeployBox>
+        </DeploayContainer>
```
Comment (markstory) [inline src/sentry/static/sentry/app/views/projectsDashboard/deploys.jsx]:
```suggestion
        </DeployContainer>
```

File: src/sentry/static/sentry/app/views/projectsDashboard/deploys.jsx [hunk 4]
```diff
@@ -70,66 +70,77 @@ class Deploy extends React.Component {
...
+  flex-shrink: 0;
+  justify-content: flex-end;
 `;
 
 class NoDeploys extends React.Component {
   render() {
     return (
-      <DeployBox p={2}>
-        <Background alignItems="center" justifyContent="center">
+      <DeploayContainer>
```
Comment (markstory) [inline src/sentry/static/sentry/app/views/projectsDashboard/deploys.jsx]:
```suggestion
      <DeployContainer>
```

File: src/sentry/static/sentry/app/views/projectsDashboard/deploys.jsx [hunk 5]
```diff
@@ -70,66 +70,77 @@ class Deploy extends React.Component {
...
+        <Background>
           <Button size="xsmall" href="https://docs.sentry.io/learn/releases/" external>
             {t('Track deploys')}
           </Button>
         </Background>
-      </DeployBox>
+      </DeploayContainer>
```
Comment (markstory) [inline src/sentry/static/sentry/app/views/projectsDashboard/deploys.jsx]:
```suggestion
      </DeployContainer>
```

**Comment by priscilawebdev** [comment]
@EvanPurkhiser unfortunately Percy screenshots don't match ...could you please check it?

**Comment by EvanPurkhiser** [comment]
Ahh yes, I see why. Thanks @priscilawebdev

**Review by markstory — APPROVED** [review]

**Pull request merged**